sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManager.scala

```diff
@@ -215,7 +215,6 @@ abstract class RapidsShuffleInternalManagerBase(conf: SparkConf, isDriver: Boole
   // so they all need to be lazy values that are executed when things are first called
```

Comment:
Need 2021 copyrights added to this file.  Sorry, missed this on initial review.

Reply:
I completely forgot. Updating.